src/cmake/Config.cmake.in

```diff
@@ -36,8 +36,11 @@
 #
 ################################################################################
 
-
 @PACKAGE_INIT@
 
+if(NOT TARGET Threads::Threads)
+  find_package(Threads REQUIRED)
+endif()
```

Comment:
can you replace it with
```suggestion
include(CMakeFindDependencyMacro)
find_dependency(Threads REQUIRED)
```

`find_dependency` is the one to be used inside the config file https://cmake.org/cmake/help/latest/module/CMakeFindDependencyMacro.html

Reply:
Learned something new - thanks. I pushed up a commit with that change.